crates/exex/exex/src/manager.rs

```diff
@@ -71,6 +76,7 @@ impl ExExHandle {
         node_head: Head,
         provider: P,
         executor: E,
+        wal: Wal,
```

Comment:
why does every exex get a new wal?
doesn't this duplicate a ton of the same data? 
this doesn't make much sense to me

Reply:
agreed, the notifications should be the same for all ExExes no matter how fast they are processing them